docs/Tutorials/demo/budget_with_farming.md

```diff
@@ -150,7 +150,7 @@ We use the following values of the fields.
 }
 ```
 
-Now, let's copy each command and paste it in your terminal. 
+Now, run each command one at a time. You can copy and paste each command in the command line in your terminal:
```

Comment:
following the guidance of https://github.com/tendermint/starport/edit/develop/docs/contributing/templates/tutorial-template.md 

I suggest that you use code fence tick marks for each command and then use plain text between 

When showing a command, explain the command first by talking about what it does. Then show the command. Then show its output in a separate output block

Reply:
That's great point and thank you for sharing the `tutorial-template.md`. That is really helpful.